test.js

```
@@ -0,0 +1,46 @@
+
+document.getElementById("output").innerHTML = "User: " + window.location.search;
+
+var arr = [1, 2, 3, 4, 5];
+for (var i = 0; i < arr.length; i++) {
+    console.log(arr[i]);
+}
+
+var x = 10;
+
+function checkAge(age) {
+    if (age = 18) { 
```

Biome

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)